Tetfuza/TetfuzaBackend/TetfuzaBackend.cs

```diff
@@ -376,23 +245,30 @@ private bool CheckMove(Coordinate center, FuzaPiece orientation)
         /// <returns></returns>
         private int CheckClear()
         {
-            int linesCleared = 0;
-            List<int> clearedIndex = new List<int>();
-            for (int i = 0; i < BOARD_HEIGHT; i++)
-            {
-                if (!(Board[i].Contains(EMPTY_CHAR) || Board[i].Contains(MOVING_CHAR)))
-                {
-                    linesCleared++;
-                    Board[i] = BoardLine('-');
-                    clearedIndex.Add(i);
-                }
-            }
-            StableFrames.Stabilize(MS_PER_FRAME * 10, _timer);
-            for (int index = 0; index < linesCleared; index++)
+            // Have board clear lines and return the amount cleared
```

Comment:
What about refactoring this into two methods? Checkclear really belongs inside the board class as a method you can test independently of the scoring figured out by the logic below